releasenotes/notes/pulse_parameter_manager_compat_with_ParameterVector-7d31395fd4019827.yaml

```diff
@@ -0,0 +1,14 @@
+---
+prelude: >
+    Compatibility of pulse parameter assignment with `ParameterVector` class.
```

Comment:
I usually leave it to the release manager to decide what makes it into the prelude.

```suggestion
```